samples/latest/HelloWeb/project.json

```diff
@@ -16,7 +16,7 @@
         "Microsoft.AspNet.Server.IIS": "1.0.0-*",

```

Comment:
Slightly unrelated, but could you also remove this dependency? We no longer produce this package.